server/src/weaverbird/backends/pypika_translator/translators/googlebigquery.py

```diff
@@ -27,6 +28,13 @@ def __init__(self, date_format: str, field: Field, alias: str | None = None):
         super().__init__("DATE_TRUNC", field, LiteralValue(date_format.upper()), alias=alias)
 
 
+class GBQParseDate(ParseDate):
```

Comment:
Was is not simpler to use ParseDate directly instead of creating this whole class ?
```python
from ... import ParseDate as GBQParseDate
```

not a blocking point, just wondering... what do you think ? or there is more that can happen here ?

Reply:
As explained in the docstring, `term` and `date_format` need to be inverted, hence the new class